model/parVarSheet.cc

```diff
@@ -40,6 +40,41 @@ namespace minsky
                                          return false;
                                        });   	
   }
+  
+  int ParVarSheet::colX(double x) const
+  { 
+	if (itemVector.empty()) return -1;
+    if ((x-offsx)<colLeftMargin[0]) return -1;
+    if ((x-offsx)<colLeftMargin[1]) return 0;
+    auto p=std::upper_bound(colLeftMargin.begin(), colLeftMargin.end(), (x-offsx));
+    size_t r=p-colLeftMargin.begin()-2+scrollColStart;
```

Comment:
Assuming scrollColStart=1, then this formula will reproduce lines 47 and 48 anyway.

Reply:
Thanks. I removed these, because they are not necessary on the tabs, the offsets offsx and offsy are enough.